lib/src/types/qr_message.dart

```diff
@@ -5,11 +5,13 @@ final class QRMessage {
   final String pairingId;
   final String webEncPublicKey;
   final String signPublicKey;
+  final String walletId;
   final bool isDemo;
 
   QRMessage.fromJson(Map<String, dynamic> json)
       : pairingId = json['pairingId'],
         webEncPublicKey = json['webEncPublicKey'],
         signPublicKey = json['signPublicKey'],
+        walletId = json['walletId'] ?? "metamask",
```

Comment:
We can add walletId in snap and dApp QR code as well.

Reply:
We have default case to handle this.